platforms/posix/src/px4/common/main.cpp

```diff
@@ -174,6 +181,22 @@ int main(int argc, char **argv)
 		return client.process_args(argc, (const char **)argv);
 
 	} else {
+#if defined(__PX4_POSIX_MEMLOCK)
```

Comment:
```suggestion
#if defined(_POSIX_MEMLOCK)
```

Is there a reason we can't just use the define directly?

> Portability Note: POSIX.1b requires that when the mlock and munlock functions are available, the file unistd.h define the macro _POSIX_MEMLOCK_RANGE and the file limits.h define the macro PAGESIZE to be the size of a memory page in bytes. It requires that when the mlockall and munlockall functions are available, the unistd.h file define the macro _POSIX_MEMLOCK.

Reply:
No. But the correct condition should be `_POSIX_MEMLOCK>0` according to man pages, which will be repeated in code for at least twice, so I decide to declare another varable to  store the comparison.

Reply:
Done.